src/test/components/TrackThread.test.js

```diff
@@ -179,6 +179,31 @@ describe('timeline/TrackThread', function() {
     expect(getCallNodePath()).toEqual(['j', 'k', 'l']);
   });
 
+  it('can click a stack in the stack graph in inverted call trees', function() {
+    const { getState, stackGraphCanvas, thread } = setup(getSamplesProfile());
```

Comment:
Thanks for the new test !

Hosever, it doesn't seem very different to the previous one!

I believe we should switch to "inverted" mode by dispatching the action `changeInvertCallstack` and test with this state. Then we should get the same results as the previous test, once "uninverted".